library/lua/gui/widgets.lua

```diff
@@ -413,6 +413,11 @@ Label.ATTRS{
     on_click = DEFAULT_NIL,
     on_rclick = DEFAULT_NIL,
     scroll_keys = STANDARDSCROLL,
+    --
```

Comment:
the separator is fine, but maybe put it above the `scroll_keys` line to keep all the scroll params together.

Reply:
did that.

The parameter was also missing from the docs - I've added it.